src/sentry/static/sentry/app/views/organizationGroupDetails/header.tsx

```diff
@@ -300,7 +321,22 @@ export {GroupHeader, TAB};
 
 export default withApi(GroupHeader);
 
+const TitleWrapper = styled('div')`
+  display: flex;
+  line-height: 24px;
+`;
+
+const InboxReasonWrapper = styled('div')`
+  margin-left: ${space(1)};
+`;
+
 const StyledTagAndMessageWrapper = styled(TagAndMessageWrapper)`
+  line-height: 1.2;
+
+  & > div:not(:last-child) {
+    margin-right: ${space(1)};
+  }
```

Comment:
Is there a way to use grid for this instead? Otherwise if you were to add more components to this, you'd need to remember that it needs to be wrapped in a div